tools/Unix.mk

```diff
@@ -616,18 +619,19 @@ ifeq ($(KCONFIG_LIB),)
   KCONFIG_NCONFIG       = kconfig-nconf Kconfig
   KCONFIG_QCONFIG       = kconfig-qconf Kconfig
   KCONFIG_GCONFIG       = kconfig-gconf Kconfig
-  KCONFIG_SAVEDEFCONFIG = kconfig-conf Kconfig --savedefconfig
+  KCONFIG_SAVEDEFCONFIG = kconfig-conf Kconfig --savedefconfig defconfig.tmp
 define kconfig_tweak_disable
 	kconfig-tweak --file $1 -u $2
 endef
 else
-  KCONFIG_OLDCONFIG     = oldconfig
-  KCONFIG_OLDDEFCONFIG  = olddefconfig
-  KCONFIG_MENUCONFIG    = menuconfig
-  KCONFIG_NCONFIG       = guiconfig
+  PURGE_MODULE_WARNING  = 2> >(grep -v "warning: the 'modules' option is not supported")
```

Comment:
why redirect 2 to empty?

Reply:
anonymous pipes :)

https://unix.stackexchange.com/questions/3514/how-to-grep-standard-error-stream-stderr